config/core/configmaps/gc.yaml

```diff
@@ -47,10 +47,40 @@ data:
     # resync period (10 hours).
     stale-revision-timeout: "15h"
 
-    # Minimum number of generations of revisions to keep before considering
-    # them for GC
+    # Minimum number of generations of non-active revisions to keep before
+    # considering them for GC.
     stale-revision-minimum-generations: "20"
 
     # To avoid constant updates, we allow an existing annotation to be stale by this
     # amount before we update the timestamp.
     stale-revision-lastpinned-debounce: "5h"
+
+
+    # ---------------------------------------
+    # V2 Garbage Collector Settings
+    # ---------------------------------------
+    #
+    # These settings are enabled via the 'responsive-revision-gc' feature flag.
+    # ALPHA NOTE: This feature is still experimental and under active development.
+
+    # Duration since revision creation before considering it for GC.
+    gc-retain-since-create-time: "48h"
+
+    # Duration since a route has pointed at the revision
+    # before considering it for GC.
+    gc-retain-since-last-active-time: "15h"
+
+    # Minimum number of non-active revisions to retain.
+    gc-min-stale-revisions: 20
```

Comment:
I'm surprised this doesn't have to be quoted 🤔

Reply:
only strings need to be quoted, but sure we can do it!